tools/db_crashtest.py

```diff
@@ -475,7 +477,8 @@ def finalize_and_sanitize(src_params):
             dest_params["use_direct_io_for_flush_and_compaction"] = 0
         else:
             dest_params["mock_direct_io"] = True
-
+    if (dest_params["manual_wal_flush_one_in"] > 0):
+        dest_params["manual_wal_flush"] = 1
```

Comment:
Combined with `"manual_wal_flush_one_in": 1000000` in `default_params`, this appears always enabled